### PR TITLE
Dicksize command

### PR DESCRIPTION
**Pull request by LBlend**
Generates a random number between 1 and 20 whenever command is called and sends it to the channel

I don't even know if this will work lol

### CODE REVIEW

**Comment by pedrofracassi** [comment]
Nonononono! The constructor should be outside of the `run` function, and what happens when you run the command should be inside of the `run` function.

File: src/commands/misc/dicksize.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+module.exports = class DickSize extends Command {
+
+  run (message) {
+    constructor (client) {
+    super(client)
+    this.name = 'dicksize'
+  }
+    
+  if const endpoint = message.channel.nsfw = True {
```
Comment (xDoges) [inline src/commands/misc/dicksize.js]: change this to if (message.channel.nsfw) {
code
}

**Comment by pedrofracassi** [comment]
Always run `npm test` before commiting. You can also `npm start fix` to fix some issues automatically.

**Comment by pedrofracassi** [comment]
**Closing due to inactivity.** Please push/comment anything if you want to reopen this.

**Pull request closed without merging**